app/window.py

```diff
@@ -873,21 +873,10 @@ def refresh(self):
   def setTextBuffer(self, textBuffer):
     app.log.info('setTextBuffer')
     #self.normalize()
-    if self.textBuffer is not None:
-      app.history.set(['files', self.textBuffer.fullPath, 'cursor'],
-          (self.textBuffer.penRow, self.textBuffer.penCol))
-    # TODO(dschuyler): Do we need to restore positions and selections here?
     self.controller.setTextBuffer(textBuffer)
     Window.setTextBuffer(self, textBuffer)
+    self.textBuffer.restoreUserHistory()
```

Comment:
Hmm I guess it's good to make it preferable. Although I would assume that the user is working at a comfortable scroll view so when they resume editing a file, the position should be saved. What about making it a setting with default as saved?